edx_solutions_api_integration/courses/utils.py

```diff
@@ -125,3 +125,8 @@ def get_user_position(course_key, **kwargs):
         data['position'] = users_above + 1
         data['completions'] = user_completions * 100
     return data
+
+
+class Round(Func):
```

Comment:
i think this should be in `edx_solutions_api_integration/utils.py` rather than `edx_solutions_api_integration/courses/utils.py`

Reply:
Addressed.